README.md

```diff
@@ -3,6 +3,57 @@
 # Task estimation
 
 # Coding standards
+
+To begin, in order for the working group to be effective and coordinated, members need to put themselves agree on several points:
+### Naming Convention
+
+A naming convention in programming is a set of rules and coding standards for choosing the rules that will make up our source code, file identifiers and documentation. The goal is to make source code easier to read with less understanding effort to do (as if we were talking the same language and which we did not need to translate in our head). It also improves the appearance code, to avoid names that are too long or abbreviations and name identifiers that would not be understandable than for the author of the code.
```

Comment:
I think there's a mistake here. "not be understandable than for the"

Reply:
Corrected ! Thank you